cas-server-core-authentication/src/main/java/org/jasig/cas/authentication/DefaultAuthenticationContext.java

```diff
@@ -0,0 +1,41 @@
+package org.jasig.cas.authentication;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link DefaultAuthenticationContext} represents a concrete
+ * implementation of the authentication context. It acts as a carrier
+ * to hold authentication sessions established during the processing
+ * of a given request, and identifies which of those sessions
+ * can be considered the primary.
+ *
+ * @author Misagh Moayyed
+ * @since 4.2
+ */
+public final class DefaultAuthenticationContext implements AuthenticationContext {
+    /**
```

Comment:
You are perfectly right. Will remove. I had a plugin configured to auto-generate docs, and it scanned everything here!